src/docs/topics/publishing-releases/README.md

```diff
@@ -0,0 +1,69 @@
+# Publishing a new Orchard Core release
+
+These notes are primarily for Orchard's core contributors to guide how to prepare a new release.
+
+## Versioning
+
+We follow [Semantic Versioning 2.0.0](https://semver.org/).
+
+## Release checklist
+
+You can copy the following checklist to a GitHub issue and tick everything as you progress. While the checklist is in a recommended order not every step depends strictly on the previous ones. "<version name>" should be replaced with the current version, e.g. "1.0.0" or "rc2".
+
+```
+### Prepare the project
+Do some housekeeping on GitHub in the [main repo](https://github.com/OrchardCMS/OrchardCore).
+
+- [ ] Close remaining issues for the version (including merging corresponding pull requests if suitable) or assign then to the next one.
+- [ ] Assign all issues that were closed for an upcoming version (including a wildcard version like "1.0.x") to this version (milestone).
+
+### Prepare the code
+Update the source so everything looks like on the new version.
```

Comment:
"everything looks like on the new version" Is there a word missing?

Reply:
No, what I want to say that change the code until it looks like what you want to have for the new version.